apps-rendering/src/themeStyles.ts

```diff
@@ -19,6 +19,8 @@ interface ThemeStyles {
 	liveblogKicker: string;
 	liveblogBackground: string;
 	liveblogDarkBackground: string;
+	deadblogKicker: string;
+	deadblogStandfirstLink: string;
```

Comment:
I'd recommend adding these to `editorialPalette` rather than `themeStyles`.

- `themeStyles` is a legacy holdover from the old templates that is unfortunately still scattered throughout our codebase. It only understands switching colour based on `Theme` (previously `Pillar`).

- `editorialPalette` is a much more comprehensive way of selecting colours based on the full `Format` object, and mirrors the functionality of `decidePalette` (we'll hopefully be able to fold them together at some point).

Thinking about where they'd fit into `editorialPalette`:

- `deadblogStandfirstLink`: I think we follow the convention already laid down by `decidePalette`, and call this `text.standfirstLink`.
- `deadblogKicker`: I'm less clear here because there appears to be no `Series` component in DCR 🤔. I'm not sure whether the `Kicker` component and `text.linkKicker` are approximate equivalents. Regardless I'd be inclined to call this `text.series`.